i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/operatingsystems-linux-centreon-monitoring-agent.md

```diff
@@ -0,0 +1,1368 @@
+| WARNINGTIME   | Thresholds                                                                                                                                            |                   |             |
+| CRITICALTIME  | Thresholds                                                                                                                                            |                   |             |
+| WARNINGTOTAL  | Thresholds                                                                                                                                            |                   |             |
+| CRITICALTOTAL | Thresholds                                                                                                                                            |                   |             |
+| EXTRAOPTIONS  | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles).      | --verbose         |             |
+
+</TabItem>
+<TabItem value="Swap" label="Swap">
+
+| Macro        | Description                                                                                                                                      | Valeur par défaut | Obligatoire |
+|:-------------|:-------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| WARNING      | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                                                                  |                   |             |
+| CRITICAL     | Threshold, can be 'usage' (in Bytes), 'usage-free' (in Bytes), 'usage-prct' (%)                                                                  |                   |             |
+| EXTRAOPTIONS | Any extra option you may want to add to the command (a --verbose flag for example). Toutes les options sont listées [ici](#options-disponibles). |                   |             |
+
+</TabItem>
+<TabItem value="Systemd-Journal" label="Systemd-Journal">
+
+| Macro           | Description                                                                                                                                                     | Valeur par défaut | Obligatoire |
+|:----------------|:----------------------------------------------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| SINCE           | Defines the amount of time to look back at messages. Can beminutes (ie 5 "minutes ago") or 'cache' to use the timestamp from last execution. (default: 'cache') | cache             |             |
```

Comment:
```suggestion
| SINCE           | Defines the amount of time to look back at messages. Can beminutes (ie 5 "minutes ago") or 'cache' to use the timestamp from last execution. | cache             |             |
```